common/config/rush/command-line.json

```diff
@@ -92,7 +92,7 @@
       "summary": "Build the Storybook",
       "description": "Build the Storybook",
       "safeForSimultaneousRushProcesses": false,
-      "shellCommand": "(cd apps/storybook && rushx storybook:baseline)"
+      "shellCommand": "(cd apps/storybook && rushx storybook:baseline:chromatic)"

```

Comment:
@craigpalermo, not familiar with rush. But what's this change after your suggested `rush change`?

Reply:
I think that's a change @sdalonzo made, I think it should go away if you rebase on `master`